docs/release-notes/2.15.0.asciidoc

```diff
@@ -0,0 +1,63 @@
+:issue: https://github.com/elastic/cloud-on-k8s/issues/
+:pull: https://github.com/elastic/cloud-on-k8s/pull/
+
+[[release-notes-2.15.0]]
+== {n} version 2.15.0
+
```

Comment:
Should we add information about the role mapping fix in 8.16, or is that not strictly related to this ECK release?